packages/amplify-auth-sdk/src/stores/secure-store.js

```diff
@@ -19,19 +26,68 @@ export default class SecureStore extends FileStore {
 	/**
 	 * Loads the `keytar` library and initializes the token file.
 	 *
-	 * @param {Object} [opts] - Various options.
+	 * @param {Object} opts - Various options.
+	 * @param {String} opts.homeDir - The path to the home directory containing the `lib`
+	 * directory where `keytar` is located.
 	 * @param {String} [opts.secureServiceName="Axway AMPLIFY Auth"] - The name of the consumer
      * using this library.
 	 * @access public
 	 */
 	constructor(opts = {}) {
+		if (!opts || typeof opts !== 'object') {
+			throw E.INVALID_ARGUMENT('Expected opts to be an object');
+		}
+
+		const { homeDir } = opts;
+		if (!homeDir || typeof homeDir !== 'string') {
```

Comment:
This now being required throws the following on an `amplify auth logout --all`, the logout command needs to pull homedir from this call [here](https://github.com/appcelerator/amplify-tooling/blob/a673f306e53602d610d8d6a9485c095cf351f33c/packages/amplify-cli-auth/src/commands/logout.js#L22-L27) and pass it in [here](https://github.com/appcelerator/amplify-tooling/blob/a673f306e53602d610d8d6a9485c095cf351f33c/packages/amplify-cli-auth/src/commands/logout.js#L29-L34)

`TypeError: Secure store requires the home directory to be specified`

We might need to audit where else might hit this path in the auth cli, and other clis to ensure those are updating as this is a breaking change

Reply:
Fixed!